hack/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
```

Comment:
Cloud you please add a short description, why this file is here?
Folder name `hack` feels kind of fishy 😅

Reply:
Done :)